src/OnboardingSPA/components/LivePreview/BlockPreview/stylesheet.scss

```diff
@@ -13,13 +13,20 @@ $main-color-light-rgb: var(--nfd-onboarding-white-rgb);
 			max-width: 320px;
 			max-height: 320px;
 		}
-		
+
 		&-large {
 			width: 70%;
-			height: 70%;
+			min-height: 90vh;
 			overflow: hidden;
-			max-width: 925px;
-			max-height: 925px;
+			position: relative;
+			align-items: center;
+			border: 1px solid #e3dfdf;
```

Comment:
Could we add the gray shade `#e3dfdf` to branding variables too?

Reply:
added this to the hardcode removal task

Reply:
As discussed, these values shall be consolidated altogether as brand variables separately.